scalableminds/webknossos
# Link to View mode from Annotation mode info tab

### URL of deployed dev instance (used for testing):
- https://linktoviewmode.webknossos.xyz

### Steps to test:
- open annotation of a dataset
- in info tab, dataset name should be link
- open private dataset’s annotation with sharing token while logged out
- view link should *not* include the token, clicking should yield error, external user is supposed to just see it via the annotation

### Issues:
- fixes #5259
related to #4772 (may be combined with settings link there)

------
